tilings/algorithms/__init__.py

```diff
@@ -35,4 +43,10 @@
     "RowColSeparation",
     "SubclassVerificationAlgorithm",
     "guess_obstructions",
+    "get_col_info",
+    "slide_column",
+    "slidable_pairs",
+    "slide_assumption",
+    "gp_slide",
+    "gp_slide_inverse",
```

Comment:
Most of these functions are too specific to the sliding stuff. I think we should think about creating a `Sliding` class for containing the algorithm and these methods (more akin to the way the other algorithms are done).